compose/bin/check-dependencies

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
The shebang should be like all of the other helper scripts: `#!/usr/bin/env bash`